matlab/test/Ice/optional/ClientPrivate.ice

```diff
@@ -12,12 +12,12 @@ module Test
 //
 // The server doesn't know this class.
 //
-class D : B
-{
-    string ds;
-    tag(990) StringSeq? seq;
-    tag(1000) A? ao;
-}
+// class D : B
+// {
+//     string ds;
+//     tag(990) StringSeq? seq;
+//     tag(1000) A? ao;
+// }
```

Comment:
Shouldn't we remove this?

Reply:
Maybe, but we also have the code commented out in a couple other language mappings already, so I just followed their lead. I'm not sure if we ever planned on un-commenting it, but if it's definitely dead code, I'll remove it!

Reply:
Please remove it.

Reply:
Removed!